scripts/ci/lib.sh

```diff
@@ -1400,6 +1400,8 @@ __EOM__
     }
 }
 
+JUNIT_DIR="${ARTIFACT_DIR:-}/junit-misc"
```

Comment:
If `ARTIFACT_DIR` isn't set, I think this be set at root level of the file system, which we probably won't have permission to create a directory. Should this be something like `JUNIT_DIR="${ARTIFACT_DIR:-$(pwd)}/junit-misc"`? We might need to worry about it at all since (I think) we usually check if `ARTIFACT_DIR` is set properly elsewhere if it's running in CI, i.e., maybe this should be `JUNIT_DIR="${ARTIFACT_DIR}/junit-misc"`

Reply:
yeah, it is a good point and bug waiting to happen.